pkg/cmd/root/help_topic.go

```diff
@@ -134,10 +134,20 @@ var HelpTopics = []helpTopic{
 			array, fields from an object, create a new array, and more. The %[1]sjq%[1]s utility does not need
 			to be installed on the system to use this formatting directive. When connected to a terminal,
 			the output is automatically pretty-printed. To learn about jq query syntax, see:
-			<https://jqlang.github.io/jq/manual/>
+			<https://jqlang.github.io/jq/manual/>.
+			
+			In addition to the built-in jq functions, the following template functions are also available:
+			- %[1]stimeago%[1]s: renders a timestamp as relative to now
+			- %[1]stimefmt(<format>)%[1]s: formats a timestamp using Go's %[1]sTime.Format%[1]s function
+
+			You can prepend jq module paths using %[1]s--library-path%[1]s to the following defaults:
+			1. ~/.jq
+			2. $ORIGIN/../gh
+			3. $ORIGIN/..
```

Comment:
It might be worth linking to the specific section of the `jq` manual that talks about this; and/or defining what `$ORIGIN` relates to here.

- https://jqlang.org/manual/#modules
 - > For paths starting with `$ORIGIN/`, the directory where the jq executable is located is substituted for `$ORIGIN`.